examples/powertools-examples-sqs/src/main/java/org/demo/sqs/SqsMessageSender.java

```diff
@@ -0,0 +1,78 @@
+
+package org.demo.sqs;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.ScheduledEvent;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.joda.JodaModule;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import software.amazon.awssdk.http.urlconnection.UrlConnectionHttpClient;
+import software.amazon.awssdk.services.sqs.SqsClient;
+import software.amazon.awssdk.services.sqs.model.MessageAttributeValue;
+import software.amazon.awssdk.services.sqs.model.SendMessageBatchRequest;
+import software.amazon.awssdk.services.sqs.model.SendMessageBatchRequestEntry;
+import software.amazon.awssdk.services.sqs.model.SendMessageBatchResponse;
+import software.amazon.lambda.powertools.logging.Logging;
+import software.amazon.lambda.powertools.logging.LoggingUtils;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import java.util.stream.IntStream;
+
+import static java.util.stream.Collectors.toList;
+
+public class SqsMessageSender implements RequestHandler<ScheduledEvent, String> {
+
+    private static final Logger log = LogManager.getLogger(SqsMessageSender.class);
+
+    private static final SqsClient sqsClient = SqsClient.builder()
+            .httpClient(UrlConnectionHttpClient.create())
+            .build();
+
+    private static final Random random = new Random();
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To make your code secure, use `new SecureRandom()` to instantiate a `java.security.SecureRandom` object. Do not use `getInstance()`, `getInstanceStrong()` and `java.util.Random`.